packages/core/src/constants/app-config.json

```diff
@@ -243,6 +243,111 @@
+                    },
+                    "locationUrl": {
+                      "title": "Location Url",
+                      "type": "string",
+                      "enum": ["https://plus.codes/4RRHM6PR+MH"]
+                    },
+                    "industryType": {
+                      "title": "Industry Type",
+                      "type": "string",
+                      "enum": ["Cattle", "Vegetable", "Mining"]
+                    },
+                    "transaction": {
+                      "type": "object",
+                      "properties": {
+                        "type": {
+                          "type": "string",
+                          "enum": ["inv"]
+                        },
+                        "consignmentNumber": {
+                          "title": "Consignment Number",
+                          "type": "string"
+                        },
+                        "documentURL": {
+                          "title": "Document Url",
+                          "type": "string"
+                        }
+                      }
+                    },
+                    "livestockIds": {
+                      "type": "array",
+                      "title": "Livestock IDs",
+                      "uniqueItems": true,
+                      "items": {
+                        "type": "string",
+                        "enum": ["9359502000010", "NH020188LEJ00005", "NH020188LEJ00008", "NH020188LEJ00012"]
+                      }
+                    }
+                  },
+                  "required": ["sender", "receiver", "livestockIds"]
+                },
+                "initialData": {},
+                "className": "json-form",
+                "style": { "margin": "auto", "paddingTop": "40px", "width": "80%" }
+              }
+            },
+            {
+              "name": "CustomButton",
+              "type": "Submit",
+              "props": {
+                "style": { "textAlign": "center", "paddingTop": "20px" }
+              }
+            }
+          ],
+          "services": [
+            {
+              "name": "processTransactionEvent",
+              "parameters": [
+                {
+                  "vckit": {
+                    "vckitAPIUrl": "http://localhost:3332",
+                    "issuer": "did:web:2dc7-115-73-168-48.ngrok-free.app"
+                  },
+                  "dpp": {
+                    "context": ["https://dpp-json-ld.s3.ap-southeast-2.amazonaws.com/transaction-event-ld.json"],
+                    "renderTemplate": [
+                      { "template": "<p>Render dpp template</p>", "@type": "WebRenderingTemplate2022" }
+                    ],
+                    "type": ["TransactionEventCredential"],
+                    "dlrLinkTitle": "Livestock Passport",
+                    "dlrIdentificationKeyType": "nlisid",
+                    "dlrVerificationPage": "http://localhost:3000/verify"
+                  },
```

Comment:
The transaction event does not issue the dpp (digital product passport). It just issue the transaction event VC.